app/src/main/res/layout/fragment_sust_details.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/fragment_sust_details"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:gravity="top"
+    tools:context=".ui.Sustainability.SustListFragment">
+
+    <!-- TODO: Update blank fragment layout -->
+    <ImageView
+        android:layout_width="match_parent"
+        android:layout_height="194dp"
+        android:scaleType="centerCrop"
+        android:src="@drawable/cascadia_banner"
+        android:contentDescription=""/>
+
+    <!-- content -->
+
+    <TextView
+        android:name="@+id/sust_details_name"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingVertical="6dp"
+        android:textSize="36dp"
+        android:text="" />
+    <TextView
```

Comment:
I'll have to remember that review strips html in addition to using markdown.  I should have used use &amp;lt;string ...&amp;gt; instead of &lt;string ...&gt;  
[Some Useful HTML Character Entities](https://www.w3schools.com/html/html_entities.asp#adngin-mid_content-0)